### PR TITLE
Log “experience” changes

### PR DESCRIPTION
**Pull request by 0x2b3bfa0**
Part of https://github.com/iterative/terraform-provider-iterative/issues/481

### CODE REVIEW

File: task/aws/task.go [hunk 1]
```diff
@@ -93,145 +93,141 @@ type Task struct {
...
 	}
-	logrus.Info("Reading Bucket...")
+	logrus.Info("[4/9] Reading Bucket...")
 	if err := t.Resources.Bucket.Read(ctx); err != nil {
 		return err
 	}
-	logrus.Info("Reading Credentials...")
-	if err := t.DataSources.Credentials.Read(ctx); err != nil {
-		return err
-	}
```
Comment (casperdcl) [inline task/aws/task.go:169]: is this not needed?

File: task/k8s/task.go [hunk 2]
```diff
@@ -109,51 +109,51 @@ func (t *Task) Create(ctx context.Context) error {
...
+		logrus.Info("[6/7] Deleting ephemeral Job to upload directory...")
 		if err := t.Resources.Job.Delete(ctx); err != nil {
 			return err
 		}
 
 		os.Unsetenv("TPI_TRANSFER_MODE")
 	}
 
-	logrus.Info("Creating Job...")
+	logrus.Info("Creating [7/7] Job...")
```
Comment (casperdcl) [inline task/k8s/task.go:132]:
```suggestion
	logrus.Info("[7/7] Creating Job...")
```

File: task/k8s/task.go [hunk 3]
```diff
@@ -167,20 +167,20 @@ func (t *Task) Delete(ctx context.Context) error {
...
 			return err
 		}
-		logrus.Info("Downloading Directory...")
+		logrus.Info("[3/7] Downloading Directory...")
 		if err := t.Pull(ctx, t.Attributes.Directory, t.Attributes.DirectoryOut); err != nil {
 			return err
 		}
 
-		logrus.Info("Deleting ephemeral Job to retrieve directory...")
+		logrus.Info("[3/6] Deleting ephemeral Job to retrieve directory...")
```
Comment (casperdcl) [inline task/k8s/task.go:183]:
```suggestion
		logrus.Info("[4/7] Deleting ephemeral Job to retrieve directory...")
```

File: task/k8s/task.go [hunk 4]
```diff
@@ -189,19 +189,19 @@ func (t *Task) Delete(ctx context.Context) error {
 		os.Unsetenv("TPI_PULL_MODE")
 	}
 
-	logrus.Info("Deleting Job...")
+	logrus.Info("[4/7] Deleting Job...")
```
Comment (casperdcl) [inline task/k8s/task.go:192]:
```suggestion
	logrus.Info("[5/7] Deleting Job...")
```

File: task/k8s/task.go [hunk 5]
```diff
@@ -189,19 +189,19 @@ func (t *Task) Delete(ctx context.Context) error {
...
 	if err := t.Resources.Job.Delete(ctx); err != nil {
 		return err
 	}
-	logrus.Info("Deleting PersistentVolumeClaim...")
+	logrus.Info("[5/7] Deleting PersistentVolumeClaim...")
```
Comment (casperdcl) [inline task/k8s/task.go:196]:
```suggestion
	logrus.Info("[6/7] Deleting PersistentVolumeClaim...")
```

File: task/k8s/task.go [hunk 6]
```diff
@@ -189,19 +189,19 @@ func (t *Task) Delete(ctx context.Context) error {
...
 	if err := t.Resources.PersistentVolumeClaim.Delete(ctx); err != nil {
 		return err
 	}
-	logrus.Info("Deleting ConfigMap...")
+	logrus.Info("[6/7] Deleting ConfigMap...")
```
Comment (casperdcl) [inline task/k8s/task.go:200]:
```suggestion
	logrus.Info("[7/7] Deleting ConfigMap...")
```

**Review by casperdcl — CHANGES_REQUESTED** [review]
cache invalidation & naming things

File: task/aws/task.go (hunk 1 above)
Comment (0x2b3bfa0) [reply task/aws/task.go:169]: It's a duplicate. See the next lines. 🙈

**Comment by 0x2b3bfa0** [comment]
🙈 🍵 ♟️

**Review by casperdcl — APPROVED** [review]

**Pull request merged**